samza-core/src/main/java/org/apache/samza/zk/ZkLeaderElector.java

```diff
@@ -105,7 +91,7 @@ public boolean tryBecomeLeader() {
     if (index == 0) {
       isLeader.getAndSet(true);
       LOGGER.info(zLog("Eligible to become the leader!"));
-      zkLeaderElectorListener.onBecomingLeader(); // inform the caller
+      leaderElectorListener.onBecomingLeader(); // inform the caller
```

Comment:
I don't think it matters much, but the caller may be in a different place from where the object was instantiated.